web/src/components/core/Page.tsx

```diff
@@ -61,7 +61,7 @@ type SectionProps = {
   /** Elements to be rendered in the section footer */
   actions?: React.ReactNode;
   /** As short as possible yet as much as needed text for describing what the section is about, if needed */
-  description?: string;
+  description?: React.ReactNode;
```

Comment:
I do not get why this change is needed. Even common message does not help

> small core/Page internal improvements

Reply:
Funny! The time I decided to go for short commit messages you catch me! :sweat_smile: 

But you're right, what a useless commit message. This change is needed to make possible passing not only a string but a JSX node. Is you have a look, I needed to pass a paragraph (`<p>`) as a section description in order to apply some styles (font-size, if I remember correctly).

Hope you got it now.

In any case, the core/Page component deserves more love too. But after the storage rewrite. With a bit of luck, we can drop the "value" param, only used in the storage world.

Reply:
ok, now it makes more sense. Maybe also it deserve a note in jsdoc above element.

Reply:
I'd improve the prop description mentioning that a React node is accepted, althoug looking at the type should be enough for someone familiar with React types, which I think it's not your case yet :)

Thanks!